EnglishFilter/sections/antiadblock.txt

```diff
@@ -550,6 +550,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! seg-ashort1a-ma.tk - Anti-AdBlock 
+@@||seg-ashort1a-ma.tk/libs/advertisement.js
```

Comment:
Where is this rule needed? I don't see the usefulness of this rule.

Reply:
Aha, I knew it.

Reply:
Sir, this website: `https://seg-ashort1a-ma.tk/`.

Reply:
Ok. Please move it to the `content_blockers.txt` with task comment.
Also add a domain to the common scriptlet rule `! anti-adblock 'adBlockDetected'.

Reply:
No no @zloyden

Reply:
It's also needed for All AdGuard Products. This `https://seg-ashort1a-ma.tk/` and `https://seg-ashort1a-ma.tk/Er/v4cw4R6b` are different IDK how, check the site's content and UI.

Reply:
Weird site. Let me double check it again.

Reply:
It seems your rule is ok here. I agree, thanks for the explanation.

Reply:
@zloyden Isn't there a way to make the site think that I have click the link? Because After circumventing all anti-adblock, it says `please click here to verify that you are a human`, when clicking on it it brings to some site then it unlocks the real original link.